game-core/src/main/java/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -165,129 +163,100 @@ private void createAndSaveStats(final boolean showPhaseStats) {
       // extended stats covers stuff that doesn't show up in the game stats menu bar, like custom resources or tech
       // tokens or # techs, etc.
       final IStat[] statsExtended = statPanel.getStatsExtended(gameData);
```

Comment:
It looks like `stats` and `statsExtended` can be moved below where you moved `alliances`, `players`, etc.  Is there something not obvious preventing that?